contributing-guides/style-guide.md

```diff
@@ -67,3 +67,12 @@ Keep the following guidelines in mind when choosing tokens:
 
 In general, tokens should make it as intuitive as possible
 to figure out how to use the command and fill it in with values.
+
+More technical wording on description lines should use the `backtick` syntax.
+Use backticks on the following:
+
+1. Paths, ex. `package.json`, `/etc/package.json`.
+2. Extensions, ex. `.dll`.
+3. Commands, ex. `ls`.
+4. Keyboard commands, ex. `CTRL+ALT+DEL`.
+5. Port numbers, ex. `8080`.
```

Comment:
I disagree strongly. Port numbers do not feel right in backticks.

Reply:
Agreed. I'll update it.